test/winter_onboarding_2021/fleet_management/views/cab_test.clj

```diff
@@ -0,0 +1,11 @@
+(ns winter-onboarding-2021.fleet-management.views.cab-test
+  (:require [clojure.test :refer [deftest is testing]]
+            [clojure.walk :as walk]
+            [winter-onboarding-2021.fleet-management-service.views.cab :as cab]))
+
+(deftest add-cab-view
+  (testing "Should have alert success bubble after adding a valid cab"
```

Comment:
Please add the test body.

Reply:
@shhivam This is not done yet. Will you be adding test body?

Reply:
@shhivam will you add test body?

Reply:
I need to figure out a way to test Hiccup forms. Might take some time @yogirajh007